0-quickie-main/pom.xml

```diff
@@ -3,54 +3,47 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <groupId>org.pinkcrazyunicorn</groupId>
+        <groupId>org.pinkcrazyunicorn.quickie</groupId>
```

Comment:
Das wurde hier überall geändert - sollten wir dann auch die ganzen Klassen in packages `org.pinkcrazyunicorn.quickie` anstelle von `org.pinkcrazyunicorn` verschieben?